server/src/main/java/com/warehousepro/service/ProductService.java

```diff
@@ -33,17 +34,15 @@ public class ProductService {
   public ProductResponse createProduct(CreateProductRequest request){
      Product product = productMapper.toProduct(request);
 
+     product.setCreatedAt(LocalDate.now());
```

Comment:
We don't need to initial `created at` and `updated at`
Check this: https://stackoverflow.com/questions/49954812/how-can-you-make-a-created-at-column-generate-the-creation-date-time-automatical